src/Illuminate/Routing/Router.php

```diff
@@ -427,15 +427,14 @@ public function apiSingletons(array $singletons, array $options = [])
      */
     public function apiSingleton($name, $controller, array $options = [])
     {
-        $only = ['show', 'update', 'destroy'];
+        $only = ['store', 'show', 'update', 'destroy'];
```

Comment:
We previously had some singleton-specific overrides in `getResourceMethods` that were applied _after_ `only`, leading to unexpected behavior.

Now, the `creatable` and `destroyable` methods just expand the defaults that are passed to `getResourceMethods`, which can then be reduced by `only`.

We need to specify `store` here so that it will be included when using `creatable`. It's safe to include even when `creatable` is not called because `only` can only reduce the defaults, but not expand them.